core/config/project_settings.h

```diff
@@ -117,7 +117,7 @@ class ProjectSettings : public Object {
 	static void _bind_methods();
 
 public:
-	static const int CONFIG_VERSION = 4;
+	static const int CONFIG_VERSION;
```

Comment:
Not an expert, but if I have to guess, this is needed because the templates might use this value so if there are 2 templates instantiation that uses this then we have declared 2 of these around? IDK.

Reply:
`CONFIG_VERSION` change isn't relevant at all for this PR.